app/src/lib/branches/service.ts

```diff
@@ -134,33 +128,18 @@ function mergeBranchesAndPrs(
 	if (pullRequests) {
 		contributions.push(
 			...pullRequests
-				.filter((pr) =>
-					remoteBranches
-						? !remoteBranches.some((rb) => isBranchNameMatch(pr.targetBranch, rb.name))
-						: false
-				)
+				.filter((pr) => !contributions.some((cb) => pr.sha == cb.sha))
 				.map((pr) => {
 					return new CombinedBranch({ pr });
 				})
 		);
 	}
 
-	// deduplicate - don't show branches that point to the same SHA
-	// prioritize PRs, then remote branches, then local only
-	const deduped = contributions.filter(
-		(value, index, self) => self.findIndex((v) => v.sha === value.sha) === index
-	);
```

Comment:
We no longer need to dedup because we're doing a better job of checking which branches are already accounted for